code_examples/sdk_examples/package.json

```
@@ -13,7 +13,7 @@
     "test": "ts-node src/test.ts"
   },
   "dependencies": {
-    "@kiltprotocol/sdk-js": "0.35.0",
+    "@kiltprotocol/sdk-js": "1",
```

```suggestion
    "@kiltprotocol/sdk-js": "^1.0.0",
```

What you had wasn't wrong, but this is better

---

Fixed at: https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29